javascript/ql/src/semmle/javascript/Functions.qll

```diff
@@ -158,6 +158,40 @@ class Function extends @function, Parameterized, TypeParameterized, StmtContaine
     result = getAReturnStmt().getExpr()
   }
 
+  /**
+   * 
+   * Functions can sometimes return without returning a value, in which case they
```

Comment:
Please see the [QLDoc Style Guide](https://wiki.semmle.com/display/IN/QLDoc+Style+Guide) (internal link) for guidelines on how to write doc comments; the other doc comments in this file provide concrete examples.